transcendence-app/seeds/002_seed_chatrooms.ts

```diff
@@ -20,7 +20,9 @@ let startTime = new Date(Date.now());
 
 const createRandomChatroomMessage = (chatroomId: string, userId: string) => {
   const oldDate = startTime;
-  startTime = new Date(startTime.setSeconds(1 + startTime.getSeconds()));
+  startTime = new Date(
+    startTime.setMilliseconds(1 + startTime.getMilliseconds()),
```

Comment:
why just one millisecond of difference?

Reply:
While seeding the chat messages, some got created far in the future, and if we write new ones in a chatroom right after and then refresh the page, the new ones would get displayed on top of the seeded ones.
I thought it was because of some pagination bug and spent some time to figure out why it happened :sweat_smile:.